pytest_sugar.py

```diff
@@ -362,9 +365,6 @@ def pytest_runtest_logreport(self, report):
             elif self.reached_last_column_for_test_status():
                 self.begin_new_line(report, print_filename=False)
 
-            rep = report
-            res = pytest_report_teststatus(report=report)
-            cat, letter, word = res
```

Comment:
@Frozenball 
This seems to still be not necessary.
Should I create a new PR for this?!